aroma/classification.py

```diff
@@ -0,0 +1,110 @@
+# CHANGES
+# -------
+# Log of changes as mandated by the original Apache 2.0 License of ICA-AROMA
+#
+#   * Drop ``runICA`` and ``register2MNI`` functions
+#   * Base ``classifier`` on Pandas, and revise signature (``predict(X)``)
+#     to make it more similar to scikit learn
+#   * Return classification labels directly on ``predict``
+#
+"""Classification functions for ICA-AROMA."""
+import logging
+
+import numpy as np
+
+LGR = logging.getLogger(__name__)
+
+# Define criteria needed for classification (thresholds and
+# hyperplane-parameters)
+THR_CSF = 0.10
+THR_HFC = 0.35
+HYPERPLANE = np.array([-19.9751070082159, 9.95127547670627, 24.8333160239175])
+
+
+def hfc_criteria(x, thr_hfc=THR_HFC):
```

Comment:
```suggestion
def hfc_criterion(x, thr_hfc=THR_HFC):
```

Since it's just one criterion.